### DISCUSSION TITLE
润出去的人，终究脱脂不能

### DISCUSSION THREAD

**Discussion by Evcklli** [The-Run-Philosophy-Organization/run#278] [General]
中国人被tg统治，已经是宽待它们了，在我看来，tg限制中国人，洗脑，限制移民，客观上防止了黄祸发生，我看这个要喊万岁
但是总有基因突变体，要所谓润，但是请记住，恁都是，恁永远都是

**Comment by SunsetMkt** [comment]
种族歧视不可取

**Reply by leon-fudan** [reply]
先把文字捋顺了再来发吧。不知所云。

**Comment by asksuperboy** [comment]
就一殖畜，跪舔洋人几把却被人家当狗耍的三流货色

**Reply by Evcklli** [reply]
骂谁呢？上面的话看懂了吗
